main.tf

```diff
@@ -1,13 +1,254 @@
-resource "random_integer" "example" {
-  count = module.this.enabled ? 1 : 0
+data "aws_caller_identity" "current" {}
+data "aws_region" "current" {}
 
-  min = 1
-  max = 50000
-  keepers = {
-    example = var.example
+locals {
+  enabled = module.this.enabled
+
+  security_group_enabled = local.enabled && var.create_security_group
+  s3_bucket_enabled      = local.enabled && var.create_s3_bucket
+  iam_role_enabled       = local.enabled && var.create_iam_role
+  account_id             = data.aws_caller_identity.current.account_id
+  region                 = data.aws_region.current.name
+}
+
+module "s3_label" {
+  source  = "cloudposse/label/null"
+  version = "0.25.0"
+  enabled = local.s3_bucket_enabled
+
+  attributes = ["s3"]
+
+  context = module.this.context
+}
+
+module "sg_label" {
+  source  = "cloudposse/label/null"
+  version = "0.25.0"
+  enabled = local.security_group_enabled
+
+  attributes = ["sg"]
+
+  context = module.this.context
+}
+
+module "iam_label" {
+  source  = "cloudposse/label/null"
+  version = "0.25.0"
+  enabled = local.iam_role_enabled
```

Comment:
add an empty line b/w version and enabled

Reply:
Updated